pkg/network/macvtap.go

```diff
@@ -0,0 +1,38 @@
+package network
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/network/cni"
```

Comment:
internal imports should be in the last group

Reply:
Done.